### PR TITLE
Add initial code

### PR DESCRIPTION
**Pull request by gortazar**
To be reviewed for potential security issues.

### CODE REVIEW

**Comment by javieraviles** [comment]
Seems like anyone can comment here

**Comment by Rubru94** [comment]
More comments

File: code.js [hunk 1]
```diff
@@ -0,0 +1 @@
+some code
```
Comment (gortazar) [inline code.js]: This does not seems like Javascript to me :)

Comment (gortazar) [reply code.js]: Yes, this should be a comment!!

**Pull request merged**